less/tiles.less

```diff
@@ -0,0 +1,100 @@
+.tile-row {
+  display: flex;
+
+  & > .tile-container {
```

Comment:
this creates a .css rule like
`.tile-row > .tile-container` that only affects direct direct children of .tile-row
we usually have a more relaxed rule that provides better flexibility
`.tile-row .tile-container` that means that affects all the descendants